dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -87,3 +87,6 @@ sslRef$Ref=SSL reference
 jwkEndpointUrl=JSON web key(JWK) end point URL
 jwkEndpointUrl.desc=Specifies a JWK end point URL.
 
+useSystemPropertiesForHttpClientConnections=Use Java system properties for HTTP client connections.
+useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections.  Set this property to true if you want the connections to use the http* and javax.* system properties.
```

Comment:
Remove extra spaces in "connections.  Set".
Do  http* and javax.* need to be consistent in "http* and javax.* system properties"? Does http* need a ".", or does "javax.*" need to have the "." removed?

These comments apply to the two other versions of the metatype.properties file.

Reply:
fixed!